environment/splatoon.env

```diff
@@ -0,0 +1,6 @@
+MONGO_URI=mongodb://mongodb:27017/pretendo_account?replicaSet=rs
+PN_SPLATOON_AUTHENTICATION_SERVER_PORT=61002
+PN_SPLATOON_SECURE_SERVER_PORT=61003
+PN_SPLATOON_ALLOW_PUBLIC_MATCHMAKING=1
+PN_SPLATOON_ACCOUNT_GRPC_HOST=account
+PN_SPLATOON_ACCOUNT_GRPC_PORT=5000
```

Comment:
Very minor but this doesn't include a newline at the end of the file, which is a standard I prefer.